src/components/CreateProfile/LocationPreferences/LocationPreferences.tsx

```diff
@@ -27,6 +27,7 @@ const LocationPreferences = () => {
           <InputFormError error={touched.country && errors.country}>
             <TextInputWithDropdown
               onChange={handleChange}
+              name="country"
```

Comment:
**nitpick:** we should have enum with inputs names to prevent copy pasting strings